fxa-client/src/config.rs

```diff
@@ -60,40 +69,54 @@ impl Config {
         issuer: String,
         jwks_uri: String,
         token_endpoint: String,
-        userinfo_endpoint: String
+        userinfo_endpoint: String,
+        client_id: String,
+        redirect_uri: String,
     ) -> Self {
-        Config {
-            content_url,
+        let fxa_config = FxAConfig {
             auth_url,
             oauth_url,
             profile_url,
             token_server_endpoint_url,
+        };
+
+        Config {
+            content_url,
             authorization_endpoint,
             issuer,
             jwks_uri,
             token_endpoint,
-            userinfo_endpoint
+            userinfo_endpoint,
+            client_id,
+            redirect_uri,
+            fxa_config
         }
     }
 
-    pub fn import_from(content_url: &str) -> Result<Self> {
+    pub fn import_from(content_url: &str, client_id: &str, redirect_uri: &str) -> Result<Self> {
         let config_url = Url::parse(content_url)?.join(".well-known/fxa-client-configuration")?;
         let resp: ClientConfigurationResponse = reqwest::get(config_url)?.json()?;
 
         let openid_config_url = Url::parse(content_url)?.join(".well-known/openid-configuration")?;
         let openid_resp: OpenIdConfigurationResponse = reqwest::get(openid_config_url)?.json()?;
 
-        Ok(Config {
-            content_url: content_url.to_string(),
+        let fxa_config = FxAConfig {
```

Comment:
I think we should go further and include the `OpenIdConfigurationResponse` fields in there too.
NIT: Maybe call it `RemoteConfig`?

Reply:
Opted to rename `OpenIdConfigurationResponse` to `RemoteConfig` and store in the `Config` object.

Reply:
But aren't they both "remote configs"? We should either:
- keep 2 very distinctive names (like *can't think of something for .well-known/fxa-client-configuration* and `OpenIdConfiguration`)
- or simply merge it into one big `RemoteConfig` struct that is the union of these 2.

Reply:
Oh I see what you meant, I thought you wanted `RemoteConfig` in addition to `FxAConfig`. I'll update it.